services/api/src/lib/circle-adapter.ts

```diff
@@ -107,40 +112,45 @@ function toCardBase(response: CircleCard): ToPaymentCardBase {
 }
 
 function toPaymentStatus(
-  status: CirclePaymentStatus,
-  verification: CirclePaymentVerification
+  status: CirclePaymentStatus | CircleTransferStatus
 ): PaymentStatus {
   let finalStatus
-  if (
-    status === CirclePaymentStatus.Failed ||
-    Object.values(CircleVerificationAVSFailureCode).includes(
-      verification?.avs
-    ) ||
-    verification?.cvv === CircleVerificationCvvStatus.Fail
-  ) {
-    finalStatus = PaymentStatus.Failed
-  } else if (
-    status === CirclePaymentStatus.Paid &&
-    Object.values(CircleVerificationAVSSuccessCode).includes(
-      verification?.avs
-    ) &&
-    verification?.cvv === CircleVerificationCvvStatus.Pass
-  ) {
-    finalStatus = PaymentStatus.Paid
-  } else if (status === CirclePaymentStatus.Confirmed) {
-    finalStatus = PaymentStatus.Confirmed
-  } else {
-    finalStatus = PaymentStatus.Pending
+  switch (status) {
+    case CirclePaymentStatus.Failed:
+      finalStatus = PaymentStatus.Failed
+      break
+    case CirclePaymentStatus.Paid:
+      finalStatus = PaymentStatus.Paid
+      break
+    case CirclePaymentStatus.Confirmed:
+      finalStatus = PaymentStatus.Confirmed
+      break
+    case CirclePaymentStatus.Pending:
+      finalStatus = PaymentStatus.Pending
+      break
+    case CircleTransferStatus.Failed:
+      finalStatus = PaymentStatus.Failed
+      break
+    case CircleTransferStatus.Complete:
+      finalStatus = PaymentStatus.Paid
+      break
+    case CircleTransferStatus.Pending:
+      finalStatus = PaymentStatus.Pending
+      break
+    default:
+      finalStatus = PaymentStatus.Pending
```

Comment:
Including transfer statuses as payment statuses, since we need the same info for creating a payment in the database, and transfer has a very similar response body.